static/js/settings_account.js

```diff
@@ -447,6 +447,15 @@ exports.set_up = function () {
         }, 5000);
     });
 
+    $("#show_my_user_profile_modal").on('click', function (e) {
+        overlays.close_overlay("settings");
+        hashchange.changehash("");
+        var user = people.get_person_from_user_id(people.my_current_user_id());
+        setTimeout(function () {
+            popovers.show_user_profile(e.target, user);
+        }, 100);
```

Comment:
The reason behind adding time out here, is that if we don't two modal(settings and user profile) crashes even if we first close settings modal.